skaffold.yaml

```diff
@@ -8,24 +8,42 @@ build:
       template: "{{.VERSION}}"
   artifacts:
   - image: valory/open-autonomy-open-aea
-    context: deployments/Dockerfiles/open_aea/
+    context: autonomy/data/agent/
     docker:
       dockerfile: Dockerfile
       buildArgs:
-        AEA_VERSION: "1.14.0"
+        AEA_VERSION: "1.14.0.post1"
         AEA_AGENT: "{{.AEA_AGENT}}"
 
 
 profiles:
-  - name: docs
+  - name: release
     build:
-      tagPolicy:
-        envTemplate:
-          template: "{{.VERSION}}"
-      artifacts:
-      - image: valory/open-autonomy-docs
-        docker:
-          dockerfile: deployments/Dockerfiles/documentation/Dockerfile
+        tagPolicy:
+          envTemplate:
+            template: "{{.VERSION}}"
+        artifacts:
+        - image: valory/open-autonomy-open-aea
+          context: autonomy/data/agent/
+          docker:
+            noCache: true
+            dockerfile: Dockerfile
+            buildArgs:
+              AEA_VERSION: "1.14.0.post1"
+              AEA_AGENT: "open_aea/my_first_aea:0.1.0:bafybeicyirm7y2ygptr5rybwznwof2k6b7fvlsvebsz4pa6cfjijva3rdu"
+        - image: valory/open-autonomy-tendermint
+          context: autonomy/data/Dockerfiles/localnode/
+          docker:
+            noCache: true
+            dockerfile: Dockerfile
+        - image: valory/open-autonomy-hardhat
+          context: autonomy/data/Dockerfiles/hardhat
+          docker:
+            noCache: true
+        - image: valory/open-autonomy-docs
+          docker:
+            noCache: true
+            dockerfile: deployments/Dockerfiles/documentation/Dockerfile
   - name: prod
```

Comment:
Why is this one empty?

Reply:
I would have to refer to @angrybayblade on that as I had not changed this.

The profile act as overrides on the base images. in the build:key and are merged into the build commands at run time.

We have separate profiles as we need to pass slightly different parameters on running.